enhancements/audit-command.md

```diff
@@ -0,0 +1,594 @@
+
+Now, if the repository has not a `PROJECT` file then, it means that the project could also be built with an SDK version < 1.0 or any other tool which is not SDK or Kubebuilder. In this way, check if the project has a `go.mod` file with the sdk dependency `github.com/operator-framework/operator-sdk`: 
+
+```go
+go 1.13
+
+require (
+	github.com/operator-framework/operator-sdk v0.17.0
+```
+
+Following how we would set the values for the above example. 
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `SDK`|
+| Language | `golang` |
+| SDK Version | `v0.17.0` |
+| Golang Version | `1.13` |
+
+**NOTE** If the `go.mod` was found but not the SDK dependency we will set the `Language` as `golang`. Also, we will always check the `go.mod` file for Golang projects to know the Golang version used.  
+
+**Gathering the info from build/Dockerfile**
+
+You will only need to check the `build/Dockerfile` when the project is not writen in Go. In this way, when the `go.mod` file cannot to be found then, it might to be an operator project using Helm or Ansible which was built with an SDK version < 1.0. In this case, we will be looking for `quay.io/operator-framework/ansible-operator` or `quay.io/operator-framework/helm-operator` images. Let's check an Ansible Operator. See the [Dockerfile](https://github.com/operator-framework/operator-sdk-samples/blob/v0.17.0/ansible/memcached-operator/build/Dockerfile): 
+
+```
+FROM quay.io/operator-framework/ansible-operator:v0.17.0
+```
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `SDK`|
+| Language | `ansible` |
+| SDK Version | `v0.17.0` |
+
+Now, see a [Dockerfile](https://github.com/operator-framework/operator-sdk-samples/blob/v0.17.0/helm/memcached-operator/build/Dockerfile) for a Helm operator project built with SDK version < 1.0. Then, note that the same fields will be obtained:
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `SDK`|
+| Language | `helm` |
+| SDK Version | `v0.17.0` |
+
+### Auditing just the latest channel heads 
+
+This option allows the user audit only the latest bundle versions (default channel heads) for each package. It can be enabled by setting `--head-only` boolean flag.
+
+By using this flag, the audit command will filter the operator bundles in the first step to download and extract just this ones. 
+
+To know the operator bundles which are the latest head we will get only the entries which has values in the `operatorbundle` table for both `csv` and `bundle` column. 
+
+### Disabling the data to perform only the checks
+
+By using the option `--data=false`, the audit command will only be gathering the operator bundle names and run the checks. All other columns described above will be skipped.
+
+This option is useful for who would like to add the audit in an automated CI job for example to check the catalog.  
+ 
+### Disabling the warning(s)
+
+By using the option `--warning=false`, the audit command will only raise the errors  found.
```

Comment:
It would be better to toggle "exit on warnings" instead, since warning logs are still useful.

Reply:
Sorry, could you please clarify your idea/suggestion? Would you like to change the name or description of the flag? 
 
The motivation is allowed use `--audit` and the flags for only get as result all errors in JSON format and not the warnings which is a useful option for CI jobs.

Reply:
Let me ask a question instead: why would you want to turn off warnings, since they do not affect the outcome of the question "is a bundle valid"? Wouldn't it be better to have a flag `--err-on-warn` to have the audit fail a particular bundle if any warnings come up?

Reply:
The `--warning=false` is to address the idea:

I want to use audit to just run bundle validation and gets the errors in a CI job. I do not want to care about the warnings and recommendations at all.

Reply:
It was changed by `--error-only` which I hope to address your feedback as well. So, I am closing this one. Feel free to make other suggestion if you wish.